x-pack/plugins/cloud/server/config.ts

```diff
@@ -24,6 +24,7 @@ const configSchema = schema.object({
   cname: schema.maybe(schema.string()),
   deployment_url: schema.maybe(schema.string()),
   id: schema.maybe(schema.string()),
+  billing_url: schema.maybe(schema.string()),
```

Comment:
FMI, starting when will this be populated by cloud?

Reply:
I don't know. I asked the infra team in Slack